### PR TITLE
Faster Faker

### PR DESCRIPTION
**Pull request by evantahler**
https://mimesis.name/en/master/ is WAY faster

1M Faker Users + Purchases Before:
Screen Shot 2022-10-15 at 12 14 24 PM

1M Faker Users + Purchases Now:

Screen Shot 2022-10-15 at 11 46 43 AM

### CODE REVIEW

File: airbyte-integrations/connectors/source-faker/setup.py [hunk 1]
```diff
@@ -5,7 +5,7 @@
 
 from setuptools import find_packages, setup
 
-MAIN_REQUIREMENTS = ["airbyte-cdk~=0.1", "Faker==13.3.1"]
+MAIN_REQUIREMENTS = ["airbyte-cdk~=0.1", "mimesis==6.1.1"]
```
Comment (davinchia) [inline airbyte-integrations/connectors/source-faker/setup.py:8]: this looks like the main change

Comment (evantahler) [reply airbyte-integrations/connectors/source-faker/setup.py:8]: Yep! And the schema of the data

File: docs/integrations/sources/faker.md [hunk 2]
```diff
@@ -41,6 +41,7 @@ N/A
 
 | Version | Date       | Pull Request                                             | Subject                                                   |
 | :------ | :--------- | :------------------------------------------------------- | :-------------------------------------------------------- |
+| 0.2.0   | 2022-10-14 | [18021](https://github.com/airbytehq/airbyte/pull/18021)     | Move to mimesis for speed!                                |
```
Comment (davinchia) [inline docs/integrations/sources/faker.md:44]: Speeeeed

**Review by davinchia — APPROVED** [review]
Can't wait to try this!

**Comment by evantahler** [comment]
Note, while this version of source-faker is 2x as fast, it still isn't fast enough to saturate the worker's CPU

Screen Shot 2022-10-15 at 12 16 12 PM

**Comment by davinchia** [comment]
@evantahler interesting! Maybe we need to make faker multi-threaded?

**Comment by evantahler** [comment]
> @evantahler interesting! Maybe we need to make faker multi-threaded?

That’s next! Or another programming language :)

**Pull request merged**